packages/image_picker/image_picker_ios/example/ios/RunnerTests/ImagePickerPluginTests.m

```diff
@@ -177,6 +179,27 @@ - (void)testPickMultiImageShouldUseUIImagePickerControllerOnPreiOS14 {
             [mockUIImagePicker setSourceType:UIImagePickerControllerSourceTypePhotoLibrary]);
 }
 
+- (void)testPickImageWithoutFullMetadataPreiOS14 {
```

Comment:
Why is this test pre-14?

Reply:
I realized that it was a wrong test case, not fully covering the new flag's use case and vulnerable to false positives. I've changed it so it now verifies that there was no check for gallery authorization status.